src/sql/workbench/services/connection/browser/connectionWidget.ts

```diff
@@ -642,13 +642,29 @@ export class ConnectionWidget extends lifecycle.Disposable {
 
 		// Display the tenant select box if needed
 		const hideTenantsClassName = 'hide-azure-tenants';
-		let selectedAccount = this._azureAccountList.find(account => account.key.accountId === this._azureAccountDropdown.value);
+		let selectedAccount: azdata.Account = this._azureAccountList.find(account => account.key.accountId === this._azureAccountDropdown.value);
```

Comment:
nit - but technically these typings for the find result should be similar to  `azdata.Account | undefined` since find can fail to find an item

```suggestion
		let selectedAccount: azdata.Account | undefined = this._azureAccountList.find(account => account.key.accountId === this._azureAccountDropdown.value);
```

Reply:
So I'd rather not specify type if that should be allowed.. 🤔

Reply:
Yeah - I usually just leave the type off in situations like this and let intellisense and the compiler figure it out (since it'll automatically give it the expected type). 

Either option is fine.

Reply:
I just liked that to get intellisense working when coding.. but I'm fine either way. Will revert that.

Reply:
You should get it without that - it'll infer the types based on the return value of find. Were you not seeing that?